app/src/main/java/com/devshish/internship/presentation/ui/lyrics/LyricsViewModel.kt

```diff
@@ -33,4 +38,16 @@ class LyricsViewModel(
             }
         }
     }
+
+    fun onLikeButtonClick(searchSong: SearchSong) {
```

Comment:
How about remove searchSong: SearchSong from parameters of this function? 
So you will get it somewhere inside of this viewmodel 

(Actually, you can add song: SearchSong to constructor here, using koin, and you will avoid calling  getLyrics() from fragment)